src/__tests__/http-test.ts

```diff
@@ -1,6 +1,4 @@
-// @flow strict
-
-import zlib from 'zlib';
+import { gzipSync, deflateSync } from 'zlib';
```

Comment:
Same here, it can be done before TS conversion.

Reply:
I just reverted this change